pkgs/servers/roon-server/default.nix

```diff
@@ -32,6 +32,7 @@ stdenv.mkDerivation rec {
     sed -i '/ln -sf/ d' Server/RoonServer
     mkdir -p $out/opt
     mv * $out/opt
+    ln -s ${zlib}/lib/libz.so.1.2.11 $out/opt/RoonMono/lib/libz.so.1
```

Comment:
Roon will complain the following without this line:

`Mono: DllImport error loading library ‘/nix/store/bs5jd82aa11ny2dp97aj1xgjp1bqq7c0-roon-server/opt/RoonMono/lib/libMonoPosixHelper.so’: ‘libz.so.1: cannot open shared object file: No such file or directory’.`

Reply:
Hard-coding 1.2.11 seems too fragile; it will probably break (silently) on the next zlib update.

_I'm just accidentally passing by._

Reply:
@vcunat that's a good point, any suggestions on how to go about it? should I copy all contents from `${zlib}/lib`?

Reply:
If the code above worked for you, why not simply this instead?
```bash
     ln -s ${zlib}/lib/libz.so.1 $out/opt/RoonMono/lib/libz.so.1
```

Reply:
👍 it works too! I'll push an update, thanks @vcunat